lib/datasource/artifactory/readme.md

```diff
@@ -0,0 +1,4 @@
+This datasource returns releases from given custom _registryUrl(s)_.
+
+The target url is composed by the _registryUrl_ and the _lookupName_,
+which defaults to _depName_ when _lookupName_ is not defined.
```

Comment:
```suggestion
This datasource returns releases from given custom `registryUrl`(s).

The target URL is composed by the `registryUrl` and the `lookupName`, which defaults to `depName` when `lookupName` is not defined.
```

URL is an abbreviation and should be capitalized.
We use the one sentence per line pattern for Markdown files, so fixing that up as well.
Also we normally use `monospaced font` for any config option names.